src/quantum_gates/_utility/simulations_utility.py

```diff
@@ -215,3 +218,64 @@ def post_process_split(source_filenames: list, target_filenames: list, split: in
         np.savetxt(target_file, mean_array)
         i += split
     return
+
+def create_random_quantum_circuit(n_qubit: int, depth: int, seed_circ: int, measured_qubit: int) -> QuantumCircuit:
+    """This function generate a random circuit in Qiskit
+
+    Args:
+        n_qubit (int): Number of qubits
+        depth (int): Depth of the circuit
+        seed_circ (int): Random seed for the random generator
+        measured_qubit (int): Number of measured qubits
+
+    Raises:
+        ValueError: If there are more measured qubits than of those used in the computation is raised an error
+
+    Returns:
+        QuantumCircuit: The random Qiskit quantum circuit
+    """
+
+
+    if measured_qubit > n_qubit:
+        raise ValueError(f"More measured qubits {measured_qubit} than of those used in the computation {n_qubit}")
+
+    n_bit = n_qubit
+    circ = QuantumCircuit(n_qubit,n_bit)
+
+    circ1 = random_circuit(n_qubit, depth, seed = seed_circ, measure=False)
+    circ.compose(circ1, inplace=True)
+    circ.barrier()
+
+    for i in range(measured_qubit):
+        circ.measure(qubit = i, cbit = i)
+
+    return circ
+
+def transpile_qiskit_circuit(circ : QuantumCircuit, init_layout: list, seed: int, backend: Backend) -> QuantumCircuit:
+        """Function to transpile a circuit using the optimal option for this backend.
+
+        Args:
+            circ (QuantumCircuit): Quantum circuit to be transpiled
+            init_layout (list): initial layout of the qubit
+            seed (int): seed for the transpilation
+            backend (BackendV2): Qiskit Backend
+
+        Returns:
+            QuantumCircuit: Transpiled circuit ready for the run
+        """
+
+        t_circ = transpile(
+        circuits= circ,
+        backend = backend,
+        initial_layout=init_layout,
+        scheduling_method='asap',
+        seed_transpiler=seed
+        )
+
+        return t_circ
+
+
+    
+
+
```

Comment:
@paolodr98 Please remove the unnecessary whitespaces. There should be a single newline at the end of each file. Following conventions like this make sure that we do not have our commits polluted with changes that do not change the logic (non-changes like whitespaces or newlines).